packages/core/src/css/index.tsx

```diff
@@ -537,24 +537,19 @@ export function mapPropsToStyledComponent<P, T>(
     defaultVariant = 'primary',
     defaultVariants = [],
   } = options
-  const {
-    breakpoint,
-    sx,
-    theme,
-    variant = defaultVariant,
-    style,
-    variants,
-  } = props
-
-  // overrride the defaults with added ones; don't get rid of them altogether
-  let multipleVariants = [...defaultVariants]
+  const { breakpoint, sx, theme, variant, style, variants } = props
+
+  // Override the defaults with added ones; don't get rid of them altogether
+  let multipleVariants = [...defaultVariants, defaultVariant]
   if (variants?.length) {
-    multipleVariants = [...defaultVariants, ...variants]
+    multipleVariants = [...defaultVariants, defaultVariant, ...variants]
   }
+  // If a variant exists make it take precedence
+  if (variant) multipleVariants.push(variant)
   multipleVariants = multipleVariants.filter(Boolean)
 
   const variantStyle = css(
```

Comment:
This is no longer used, right?

Reply:
Which part are you referring to?

Reply:
Never mind, I confused `variant` with `defaultVariant`

Reply:
Oh, no, I meant this line. Shouldn't we delete `variantStyle` as a variable altogether, since it's deleted in line 595 and replaced by `multipleVariantStyle`?

Reply:
Ahhh I see. The only reason I left it was because I didn't know if it had a direct impact on any of the font family logic. I'm glad you took a look at it, if you think it's safe to remove I'll go ahead and remove it.

Reply:
Ah, I missed that. You're right. Not totally sure what to do above the font-family here. I think we can leave this as is.